closure/compiler/closure_js_library.bzl

```diff
@@ -101,13 +97,16 @@ def _closure_js_library_impl(
         deprecated_stderr_file = None,
         deprecated_ijs_file = None,
         deprecated_typecheck_file = None):
+    if not no_closure_library:
+        deps = deps + ctx.attr._closure_library_base
```

Comment:
It appears the upstream at some point switched from using an argument typically filled from `ctx.files._closure_library_base` to using `ctx.attr._closure_library_base`. But we were still using the former structure. With this change and that on line 109, the argument is no longer used, which feels wrong.

So should:
* this line use the `closure_library_base` argument (like old line 110 was) instead of the `ctx.attr` field?
* the argument be removed from the method signature (incidentally bringing it more in line with the upstream)?

Or am I missing something about why this is the way it is?